projects.yml

```diff
@@ -1246,16 +1246,22 @@ joplin:
   categories: [Database Migrations]
   platforms: [clj]
 
-clj_aws_s3:
-  name: clj-aws-s3
-  url: https://github.com/weavejester/clj-aws-s3
-  categories: [S3 Clients]
-  platforms: [clj]
-
 amazonica:
   name: amazonica
   url: https://github.com/mcohen01/amazonica
-  categories: [S3 Clients]
+  categories: [Amazon Web Services]
```

Comment:
Can you change the category to "Amazon Web Services (AWS)"? That way if someone searches for "aws" they'll get the whole category.